demo/client/backend/fabric-gateway/fabric-sdk-node_fabric-network_lib_transaction.js.patch

```diff
@@ -0,0 +1,31 @@
+--- transaction.js	2019-12-22 17:18:22.004770000 +0000
++++ node_modules/fabric-network/lib/transaction.js	1985-10-26 08:15:00.000000000 +0000
```

Comment:
I have checked the sdk (https://registry.npmjs.org/fabric-network/-/fabric-network-1.4.4.tgz) and the docker image, but I could not find the lines that this patch deletes -- i.e., `transaction.js` is already in the expected state.
Am I missing something?

Reply:
note it is a reverse patch (see Dockerfile), so the '-' are '+' . sorry for the confusion ...